app/src/main/java/com/gathering/android/event/EventComposible.kt

```diff
@@ -197,31 +196,34 @@ fun EventItem(
             Modifier
                 .background(Color.Transparent)
         ) {
-            EventImage(
-                photoUrl = event.photoUrl,
-                size = 170.dp, modifier = Modifier
-            )
-            Row(
-                modifier = Modifier.fillMaxWidth(),
-                horizontalArrangement = Arrangement.SpaceBetween,
-                verticalAlignment = Alignment.CenterVertically
-            ) {
+            event?.let { validEvent ->
+                EventImage(event = validEvent)
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                    horizontalArrangement = Arrangement.SpaceBetween,
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+                    ShowText(
```

Comment:
ShowText is not a good name for a composable. it could be CustomAlert, CustomMessage, CustomError etc

Reply:
i did change this in my previous PR that you just approved, thanks